octobot/constants.py

```diff
@@ -44,12 +45,21 @@
 DEFAULT_COMMUNITY_WS_URL = "ws://community2.octobot.cloud/"   #TODO
 OCTOBOT_COMMUNITY_URL = os.getenv("COMMUNITY_SERVER_URL", DEFAULT_COMMUNITY_URL)
 OCTOBOT_COMMUNITY_FEED_URL = os.getenv("OCTOBOT_COMMUNITY_WS_URL", f"{DEFAULT_COMMUNITY_WS_URL}cable")
-OCTOBOT_COMMUNITY_AUTH_URL = f"{OCTOBOT_COMMUNITY_URL}spree_oauth/token"
+OCTOBOT_COMMUNITY_FEED_URL = os.getenv("OCTOBOT_COMMUNITY_MQTT_URL", "iot.fr-par.scw.cloud")    #TODO
```

Comment:
do we confirm this mqtt provider for beta version?

Reply:
Yes

Reply:
Ok, I cleaned up all those constants